docs/mcp/client.md

```diff
@@ -84,6 +81,10 @@ Will display as follows:
 
 ![Logfire run python code](../img/logfire-run-python-code.png)
 
+#### Transport
+
+`MCPServerHTTP` supports both SSE (`sse`) and Streamable HTTP (`streamable-http`) servers which can be specified via the `transport` parameter.
```

Comment:
I think we can drop this section as I suggested mentioning the `transport` arg above.

It'd be nice if we could also have a Streamable HTTP example!